src/tailwind-sx/src/generate.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import fs from 'fs';
+import path from 'path';
+import prettier from 'prettier';
+import SignedSource from '@adeira/signed-source';
+
+import generateSX from './sxGenerator';
+
+// wget https://unpkg.com/tailwindcss@1.8.3/dist/tailwind.css
+const css = fs.readFileSync(path.join(__dirname, 'tailwind.css'));
+const styles = generateSX(css);
+
+const code = SignedSource.signFile(`
+/**
+ * ${SignedSource.getSigningToken()}
+ * @flow
+ *
+ * Tailwind CSS file converted into SX stylesheets
+ *
+ * @see https://tailwindcss.com/
+ * @see https://github.com/adeira/sx
+ */
+
+import * as sx from '@adeira/sx';
+
+// eslint-disable-next-line flowtype/no-weak-types
+const tailwindSX: Object = sx.create(${JSON.stringify(styles)});
```

Comment:
Would it be possible to avoid weak types like `Object` please? Some alternatives are suggested here: https://github.com/adeira/universe/blob/67e496b09730e6ed01a3f727179ff7f6b2d90cef/src/eslint-config-adeira/ourRules.js#L285-L286